pkg/security/probe/probe.go

```diff
@@ -25,6 +27,10 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/security/serializers"
 )
 
+const (
+	defaultConsumerChanSize = 50
```

Comment:
This may be too small for some consumers. For the network consumer we already have queue in place with a size of `100`, and we're seeing dropped events in deployments. I am very likely going to bump the channel on the network consumer side. If we merge this PR, I will remove this channel on the network consumer side, in which case making this value configurable per consumer would be very useful.

Reply:
Done